patch/0042-Marvell-a385-Micron-4G-flash-support.patch

```diff
@@ -0,0 +1,41 @@
+From 5e248248ac5be0541cacb45b2b4d05df3741c2c6 Mon Sep 17 00:00:00 2001
+From: gilt <gilt@marvell.com>
+Date: Mon, 22 Jul 2019 13:44:06 +0300
+Subject: [PATCH 1/4] Micron 4G flash support
+
+Change-Id: Ib4dd7094920c124c213f6b3c81140845e5baa084
```

Comment:
If this has been sent upstream, please add the URL or git hash in the Linux repository.